OpenNefia.Content/UI/Hud/IHudLayer.cs

```diff
@@ -1,9 +1,15 @@
-﻿using OpenNefia.Core.UI.Layer;
+﻿using OpenNefia.Core.Maths;
+using OpenNefia.Core.UI.Layer;
 
 namespace OpenNefia.Content.UI.Hud
 {
     public interface IHudLayer : IUiLayer
     {
         public IHudMessageWindow MessageWindow { get; }
+        public UIBox2i GameBounds { get; }
+        void ToggleBacklog(bool visible);
+        void UpdateTime();
+        void UpdateMinimap();
```

Comment:
As mentioned, I think `UpdateTime/Minimap()` should be consolidated into one `UpdateHud()`. In the future, mods would be able to add their own HUD widgets, and the clock/minimap would be a part of that system, so it doesn't make sense to have methods updating just those two widgets.

Reply:
changed in 9eb449f